hubitat-mitsubishi-mqtt.groovy

```diff
@@ -85,13 +86,16 @@ metadata {
 
         attribute 'vane', 'enum', vanePositions
         attribute 'wideVane', 'enum', wideVanePositions
+        attribute 'remoteTemperature', 'number'
 
         command 'setThermostatFanMode', [[name: 'Set Thermostat Fan Mode', type: 'ENUM', constraints: supportedThermostatFanModes]]
         command 'setThermostatMode', [[name: 'Set Thermostat Mode', type: 'ENUM', constraints: supportedThermostatModes]]
 
         command 'dry'
         command 'vane', [[name: 'Position*', type: 'ENUM', constraints: vanePositions]]
         command 'wideVane', [[name: 'Position*', type: 'ENUM', constraints: wideVanePositions]]
+        command 'setRemoteTemperature',
+                [[name: "Temperature ${getTemperatureUnit()} (0 for internal sensor)", type: 'NUMBER']]
```

Comment:
1) move the hint text to `description`
2) Naming it Temperature* makes it required. I see that you are testing for null below so we could just make it required and skip. However, I wonder if it might be better to make `null` the way that you set the sensor back to internal? If we stick with 0 let's make this required.